### PR TITLE
Gwb grid limit resolution

### PR DESCRIPTION
**Pull request by MFraters**
This adds an option to limit the resolution of gwb-grid, and applies that to the cookbook tester.

### CODE REVIEW

File: source/gwb-grid/main.cc [hunk 1]
```diff
@@ -363,7 +366,6 @@ int main(int argc, char **argv)
               number_of_threads = Utilities::string_to_unsigned_int(options_vector[i+1]);
               options_vector.erase(options_vector.begin()+static_cast<std::vector<std::string>::difference_type>(i));
               options_vector.erase(options_vector.begin()+static_cast<std::vector<std::string>::difference_type>(i));
-              --i;
```
Comment (tjhei) [inline source/gwb-grid/main.cc:366]: I am no longer convinced this is correct. I am too tired to figure out.

File: tests/CMakeLists.txt [hunk 2]
```diff
@@ -256,7 +256,7 @@ file(GLOB_RECURSE COOKBOOK_TEST_SOURCES "${CMAKE_SOURCE_DIR}/cookbooks/*/*.wb")
 foreach(test_source ${COOKBOOK_TEST_SOURCES})
   get_filename_component(test_name ${test_source} NAME_WE)
   get_filename_component(test_dir ${test_source} DIRECTORY)
-  set(TEST_ARGUMENTS "${test_dir}/${test_name}.wb\;${test_dir}/${test_name}.grid")
+  set(TEST_ARGUMENTS "${test_dir}/${test_name}.wb\;${test_dir}/${test_name}.grid\;--resolution-limit\;75")
```
Comment (tjhei) [inline tests/CMakeLists.txt:259]: 20?

**Review by tjhei — APPROVED** [review]

File: tests/CMakeLists.txt (hunk 2 above)
Comment (MFraters) [reply tests/CMakeLists.txt:259]: I am not sure 20 will be enough to ensure hitting any not huge feature in a model. 50?

File: source/gwb-grid/main.cc (hunk 1 above)
Comment (tjhei) [reply source/gwb-grid/main.cc:366]: I vote for keeping ``--i;``

File: tests/CMakeLists.txt (hunk 2 above)
Comment (tjhei) [reply tests/CMakeLists.txt:259]: deal.

**Pull request merged**